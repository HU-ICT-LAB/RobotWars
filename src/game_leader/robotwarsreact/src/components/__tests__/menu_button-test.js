import TestRenderer from "react-test-renderer";
import MenuButton from '../menu_button';

it("Renders correctly", () => {
  const renderer = TestRenderer.create(<MenuButton/>);
  expect(renderer.toJSON()).toMatchSnapshot();
});