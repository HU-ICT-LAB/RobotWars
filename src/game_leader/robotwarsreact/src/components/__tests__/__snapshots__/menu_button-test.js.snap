// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<button
  className="button"
  onClick={[Function]}
/>
`;
